### ISSUE TITLE
Crash setting loadCoreRecipes to false

### ISSUE THREAD

**Issue by tgstyle** [SleepyTrousers/EnderIO#4791]
#### Issue Description:
Setting the mod to not load the core recipes causes a crash

#### What happens:
Game dies

#### What you expected to happen:
Game lives

#### Steps to reproduce:

1. Edit the EnderIO.cfg and set B:loadCoreRecipes=false
2. Launch Game
...

____
#### Affected Versions (Do *not* use "latest"):

- EnderIO: 5.0.29
- EnderCore: 0.5.35
- Minecraft: 1.12.2
- Forge: 2739

#### Your most recent log file where the issue was present: 

https://gist.github.com/tgstyle/8907715a715d08e5ae03a212efc5d9f6

**Comment by tgstyle** [comment]
Yeah I figured that after I reported this. However for testing I moved in the default xml files, and it breaks because it is not loading the alias file. How can we use aliases? Does it load a specific file first?

**Comment by HenryLoenwind** [comment]
The core files are loaded in a specific order, when using the same file sin the user folder, the loading order is random. We recently fixed the loading to load aliases from all files before the recipes (mostly to allow user files to override the core aliases), that fixes this issue, too.

Expect a new release with this feature some time in the next week.

**Comment by tgstyle** [comment]
Awesome, Thanks!
